data/vaults/0x19D3364A399d251E894aC732651be8B0E4e85001.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "vault",
+  "comment": "Original DAI v2",
+  "hideAlways": false,
+  "depositsDisabled": true,
+  "withdrawalsDisabled": false,
+  "order": 4,
+  "apyOverride": "N/A",
+  "migrationAvailable": true,
+  "migrationContract": "0x1824df8D751704FA10FA371d62A37f9B8772ab90",
+  "migrationCall": "migrateAll(0x19D3364A399d251E894aC732651be8B0E4e85001, 0xdA816459F1AB5631232FE5e97a05BBBb94970c95)",
```

Comment:
Do we need to worry about someone getting access to IFPS then using a malicious method here? Not sure exactly how it would be done but maybe they could replace it with `approve(...)` and then transfer the token to themselves?

Reply:
Hmmm, @xgambitox could you provide some feedback here?

Tbh I don't know enough about how IPFS works to know how easy it is to add malicious code.